imports/plugins/core/taxes/lib/simpleSchemas.js

```diff
@@ -2,6 +2,16 @@ import SimpleSchema from "simpl-schema";
 
 export const Taxes = new SimpleSchema({
   _id: String,
+  /**
+   * Custom key/value data that you need to store.
+   * You'll need to extend GraphQL schemas if you
+   * want to expose any of this data through the API.
+   */
+  customData: {
+    type: Object,
+    blackbox: true,
+    optional: true
+  },
```

Comment:
This is the only actual change. All other changes were in support of adding the tests.